ci/ci.sh

```diff
@@ -4,12 +4,12 @@ echo "INFO: Run id: $RUN_ID"
 echo "INFO: Sleep seconds set for services to start: ${SLEEP:=60}"  # defaults to 60 seconds
 
 if [[ $NODE_ENV == "local_client-public" ]]; then
-  $(node node_modules/@ceramicnetwork/cli/bin/ceramic daemon --verbose --log-to-files --network dev-unstable --ethereum-rpc https://rinkeby.infura.io/v3/b6685df41e1647c4be0046dfa62a020b) &
+  $(node node_modules/@ceramicnetwork/cli/bin/ceramic daemon --verbose --log-to-files --network dev-qa --ethereum-rpc https://rinkeby.infura.io/v3/b6685df41e1647c4be0046dfa62a020b) &
```

Comment:
Can we do this yet? Will also need `js-ceramic` updates since the peerlist is still in the Ceramic code.

Reply:
Probably need to add the QA IPFS peers to a new peerlist file and update `js-ceramic` to use the new network, or at least need to coordinate with Network based on the plan for the peerlist code in Ceramic.

cc @stbrody

Reply:
yeah, you can't just set an arbitrary network, each network needs explicit support in js-ceramic, so if you want to add a new dev-qa network that will need it's own change to js-ceramic.

As for the peer connectivity, I think we should plan to remove the peerlist entirely soon, but we should have some hardcoded peerIDs in the js-ceramic codebase for our infra peers.  Much the way that IPFS ships with some hardcoded "bootstrap" peers.

Reply:
Leaving `dev-unstable` here would mean that both the `dev-qa` and `dev` envs will use the same pubsub topic. I guess that's not a huge deal.

Reply:
I think we should use `dev-qa` from here forward

Reply:
Yeah, that's cool. Will just be blocked here until the corresponding Ceramic changes are made.

Reply:
oop. sorry deleted my comment. i guess it is nice though having an env type use the same topic. 🤔 
and probably fair to keep calling it `dev-unstable` because it will still be unstable with frequent deployments and code changes

Reply:
Yeah, although sharing the same topic across environments worries me a bit. If we had a dev-like perf environment, for example, we'd want to separate the topics for sure.

Reply:
yes agree re: perf. that will need new topics.

will keep as dev-unstable for now here